test/services/sync/strategies/CSVSyncer/CSVSyncer.unit.test.js

```diff
@@ -75,5 +78,23 @@ describe('CSVSyncer', () => {
 			expect(result[1].class).to.equal('');
 			expect(result[2].class).to.equal('2b');
 		});
+
+		it('should not accept unvalid dates as user birthday', () => {
+			const falsyValues = [false, null, undefined, '', 0]
+			const misfitValues = [Symbol(), [], {}, true]
+			const wrongFormatDates = ['32.12.2000', '01.13.2000', '01.01-2000', '01/01.2000', '01-01/2000', '42', 'void']
+			const correctFormatDates = ['01.01.2000', '01-01-2000', '01/01/2000']
+
+			const result0 = falsyValues.map(f => CSVSyncer.dateValidator(f))
+			const result1 = misfitValues.map(m => CSVSyncer.dateValidator(m))
+			const result2 = wrongFormatDates.map(w => CSVSyncer.dateValidator(w))
+			const result3 = correctFormatDates.map(c => CSVSyncer.dateValidator(c))
+
+			result0.forEach(r => expect(r).to.be.equal('missing birthday value'))
+			result1.forEach(r => expect(r).to.be.equal('incorrect values, birthday must be a string'))
+			result2.forEach(r => expect(r).to.be.equal('incorrect format. Birthday must be dd.mm.yyyy or dd/mm/yyyy or dd-mm-yyyy'))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 132. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5220139)